tests/test_standard_parameter_type.py

```diff
@@ -1,20 +1,52 @@
 # ---------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
+from inference_schema.schema_decorators import input_schema, output_schema
+from inference_schema.parameter_types.standard_py_parameter_type import StandardPythonParameterType
+from inference_schema.schema_util import get_supported_versions
 
 
 class TestStandardPythonParameterType(object):
 
-    def test_standard_handling(self, decorated_standard_func):
-        standard_input = {'name': ['Sarah'], 'state': ['WA']}
-        state = {'state': ['WA']}
+    def test_standard_handling_unique(self, decorated_standard_func):
+        standard_input = {'name': ['Bill'], 'state': ['ME']}
+        state = {'state': ['ME']}
         result = decorated_standard_func(standard_input)
         assert state == result
 
-        standard_input = {'param': {'name': ['Sarah'], 'state': ['WA']}}
+        standard_input = {'param': {'name': ['Bill'], 'state': ['ME']}}
         result = decorated_standard_func(**standard_input)
         assert state == result
 
+        version_list = get_supported_versions(decorated_standard_func)
+        assert '2.0' in version_list
+        assert '3.0' in version_list
+        assert '3.1' in version_list
+
+    def test_standard_handling_list(self):
+        def decorated_standard_func(standard_sample_input, standard_sample_output):
```

Comment:
Please pull this out into a fixture like the other decorated functions, for consistency.

Reply:
Since they never get reused, having them in a fixture feels like needless confusion - fixtures + decorators named similar/the same things made it much harder to understand at first. Will change for consistency, but I don't understand why the pattern as a whole is a good one.

Reply:
In general the purpose of the pattern is to promote code reuse in the tests, so even if nothing else would reuse it yet, having it as a fixture already to be able to reuse in the future in my opinion can help to push not creating something slightly different for every test. I agree that the naming could be better though.